sdk/src/action-support.ts

```diff
@@ -557,53 +599,56 @@ class ActionHandler {
       this.streamDebug('Received stream end');
       const userReturn = this.invokeCallback('end', this.ctx);
       if (userReturn instanceof Reply) {
-        this.passReplyThroughContext(this.ctx, userReturn);
+        this.passReplyThroughContext(ctx, userReturn);
       } else {
         this.streamDebug(
           'Ignored unknown (non Reply) return value from end callback',
         );
       }
     });
 
-    /**
-     * Cancel the incoming stream of messages.
-     *
-     * @function module:kalix.Action.StreamedInContext#cancel
-     */
-    this.ctx.cancel = () => {
-      if (this.call.cancelled) {
+    ctx.cancel = () => {
+      if (call.cancelled) {
         this.streamDebug('cancel invoked when already cancelled.');
       } else {
-        this.call.cancel();
+        // FIXME: there doesn't seem to be any server-side cancel, only client-side, so what was this doing before?
+        // call.cancel();
```

Comment:
Hmmm. Tried looking into the grpc-js interfaces but I can't find anything suitable-cancel-like either.